src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -623,6 +623,22 @@ private void StepInternal(int threadId, enum_STEPKIND stepKind, enum_STEPUNIT st
             }
         }
 
+        private enum ClientId
+        {
```

Comment:
I would suggest declaring 'Unknown' as the first value. This will mean that if anyone checks `m_clientId` before the Initialize request, they will get 'Unknown' instead of 'VisualStudio'.